packages/flutter_tools/lib/src/dart/package_map.dart

```diff
@@ -17,6 +17,16 @@ Map<String, Uri> _parse(String packagesPath) {
 class PackageMap {
   PackageMap(this.packagesPath);
 
+  static PackageMap createGlobalInstance() => new PackageMap(globalPackagesPath);
```

Comment:
Actually, it's created infrequently, and since it caches the cwd when it's created (and the tool now changes the cwd at times), I'd be happier if we create a new instance each time we need one.